src/app/features/ui/navbar/navbar.component.ts

```diff
@@ -0,0 +1,40 @@
+import { ChangeDetectionStrategy, Component, inject, Input } from '@angular/core';
+import { NotificationsQuantityComponent } from './notifications-quantity.component';
+import { Router } from '@angular/router';
+import { NgIf, NgClass } from '@angular/common';
+
+@Component({
+  standalone: true,
+  imports: [NotificationsQuantityComponent, NgIf, NgClass],
+  selector: 'app-navbar',
+  template: `
+    <div class="navbar">
```

Comment:
skoro to navbar, to czemu nie element `nav`?

Reply:
Done